client/asset/eth/eth.go

```diff
@@ -1248,13 +1253,12 @@ func (w *assetWallet) PreRedeem(req *asset.PreRedeemForm) (*asset.PreRedeem, err
 // SingleLotRedeemFees is a fallback for PreRedeem that uses estimation when
 // funds aren't available. The returned fees are the RealisticWorstCase.  The
 // Lots field of the PreSwapForm is ignored and assumed to be a single lot.
-func (w *assetWallet) SingleLotRedeemFees(form *asset.PreSwapForm) (fees uint64, err error) {
-	redeemSize := form.AssetConfig.RedeemSize
-	if redeemSize == 0 {
-		g := w.gases(form.AssetConfig.Version)
-		redeemSize = g.Redeem
+func (w *assetWallet) SingleLotRedeemFees(form *asset.PreRedeemForm) (fees uint64, err error) {
```

Comment:
Seems that eth wasn't a BotWallet because of the input arg type.  Added an assertion and fixed the input type.